saritasa_invocations/alembic.py

```diff
@@ -0,0 +1,147 @@
+import collections
+import os
+
+import invoke
+
+from . import printing, python
+
+
+@invoke.task
+def run(context: invoke.Context, command) -> None:
+    """Execute alembic command."""
+    config = context.config.get("saritasa_invocations", {})
+    alembic_command = config.get("alembic_command", "-m alembic")
+    python.run_python(
+        context,
+        command=f"{alembic_command} {command}",
+    )
+
+
+@invoke.task
+def autogenerate(
+    context: invoke.Context,
+    message="",
+) -> None:
+    """Autogenerate new version with `message` title.
+
+    Similar to the `manage.py makemigration` command in Django. But you always
+    have to check generated versions before upgrade and fix it manually
+    if it's necessary.
+
+    """
+    if not message:
+        raise invoke.Exit(
+            code=1,
+            message=(
+                'Please, use `-m "Version message"` to set message '
+                "for autogenerated version."
+            ),
+        )
+    printing.print_success("Autogenerate migrations")
+    config = context.config.get("migrations_folder", {})
+    migrations_folder = config.get(
+        "migrations_folder",
+        "db/migrations/versions",
+    )
+    rev_id = str(
+        len(
+            [
+                name
+                for name in os.listdir(migrations_folder)
+                if os.path.isfile(f"{migrations_folder}/{name}")
+            ],
+        )
+        + 1,
+    )
+    rev_id = rev_id.rjust(4, "0")
+    command = (
+        f'revision --autogenerate --message "{message}" --rev-id={rev_id}'
+    )
+    run(context, command=command)
```

Comment:
Let's add further instructions for developer like:
* Verify migrations
* Remove alembic's messages